### PR TITLE
CORE-6613 fix issue when cancel is selected from filefolderselector d…

### PR DESCRIPTION
**Pull request by sri2k1us**
…ialog, it doesn't appear in the copy dialog.

### CODE REVIEW

**Comment by ianmcorvidae** [comment]
I don't know the significance of the change from `onHide` to `onDialogHide` but the rest of the changes (i.e. guarding the operation with a check that it's the appropriate button) look good to me.

File: ui/de-lib/src/main/java/org/iplantc/de/diskResource/client/views/widgets/MultiFileSelectorField.java [hunk 1]
```diff
@@ -130,23 +133,25 @@ public FileSelectDialogHideHandler(final FileSelectDialog dlg,
...
+        	if(event.getHideButton().equals(PredefinedButton.OK)) {
+	            Set<File> files = diskResourceUtil.filterFiles(dlg.getDiskResources());
+	            if (files.isEmpty()) {
+	                return;
+	            }
+	            store.addAll(files);
+	            if (userSettings.isRememberLastPath() && store.size() > 0) {
+	                userSettings.setLastPath(diskResourceUtil.parseParent(store.get(0).getPath()));
+	                eventBus.fireEvent(new LastSelectedPathChangedEvent(true));
+	            }
```
Comment (ianmcorvidae) [inline ui/de-lib/src/main/java/org/iplantc/de/diskResource/client/views/widgets/MultiFileSelectorField.java]: Though, hm. Should the "remember last path" functionality still work even when the dialog is cancelled? With this change, it looks like the path will only be remembered if "OK" is clicked, which may or may not be the intent.

Comment (sri2k1us) [inline ui/de-lib/src/main/java/org/iplantc/de/diskResource/client/views/widgets/MultiFileSelectorField.java]: Good catch...I will add logic to save last path even when cancel is clicked.

Comment (sri2k1us) [inline ui/de-lib/src/main/java/org/iplantc/de/diskResource/client/views/widgets/MultiFileSelectorField.java]: I just checked the behavior in other places, it does look like we only save last path only when user hits 'OK' button. So I will stick to that.

**Pull request merged**